tests/test_sourmash.py

```diff
@@ -2784,8 +2784,8 @@ def test_gather_abund_10_1():
 
         print(out)
         print(err)
-        assert '91.0%  100.0%      tests/test-data/genome-s10.fa.gz' in out
-        assert '9.0%   80.0%      tests/test-data/genome-s11.fa.gz' in out
+        assert '91.0%  100.0%      14.5    tests/test-data/genome-s10.fa.gz' in out
```

Comment:
I think this should right-align... am I being too perfectionest?

Reply:
See comment above in `commands.py`.

Reply:
do you mean [this comment](https://github.com/dib-lab/sourmash/pull/439/files#r174647289) or something in the actual commands.py file?

Reply:
Sorry-- not sure why that got bungled. I put in {:>9}. Is that not right-aligned?

Reply:
oh, I see - I misread things. sorry, looks good :)